Makefile

```diff
@@ -58,6 +60,7 @@ gen:
 		-p disallowAdditionalPropertiesIfNotPresent=false \
 		--git-user-id ${GIT_ORG} \
 		--git-repo-id ${GIT_REPO}/${PACKAGE_PREFIX} \
+		-t /local/templates \
```

Comment:
Given how many in-flight things are using custom templates, this should probably be adopted on its own.